README.md

```diff
@@ -72,18 +72,18 @@ Or if you don't want to trace an entire function, you can wrap the relevant part
 import pysnooper
 import random
 
-def foo():
+def foo(elapsed_time=False):
     lst = []
     for i in range(10):
         lst.append(random.randrange(1, 1000))
 
-    with pysnooper.snoop():
+    with pysnooper.snoop(elapsed_time=elapsed_time):
```

Comment:
`elapsed_time` should not be mentioned here, this section is about `with`.

Reply:
That's correct, but I'll fix the readme after we approve this PR, so you can ignore that in your review.